vortex-roaring/Cargo.toml

```diff
@@ -20,6 +20,8 @@ linkme = { workspace = true }
 croaring = { workspace = true }
 num-traits = { workspace = true }
 log = { workspace = true }
+serde = { workspace = true }
```

Comment:
Can you file a FLUP issue for vortex-array to re-export these macros so libraries don't need to take an explicit dependency?

Basically, `vortex-array/src/lib.rs` has a `pub use paste;` then the `impl_array` macro uses `$crate::paste`` instead of `paste`